test/utility/parse_opts.c

```diff
@@ -47,19 +47,27 @@ testutil_parse_opts(int argc, char *const *argv, TEST_OPTS *opts)
     opts->argv0 = argv[0];
     opts->progname = testutil_set_progname(argv);
 
+    opts->insertion_sleep_str = NULL;
+
     testutil_print_command_line(argc, argv);
 
-    while ((ch = __wt_getopt(opts->progname, argc, argv, "A:dh:b:n:o:pR:T:t:vW:")) != EOF)
+    while ((ch = __wt_getopt(opts->progname, argc, argv, "A:C:dh:I:b:n:o:pr:R:T:t:vW:")) != EOF)
```

Comment:
Nit: 1. This is outdated as I fixed the out-of-alphabetical-order args a few days ago. You'll get a conflict here.
2. We alphabetize the lower case after upper case: `R:r:`